CatoNetworks/cato-sase/ingest/parser.yml

```diff
@@ -114,3 +137,47 @@ stages:
           cato.sase.event_type: "{{json_event.output.event_type}}"
           cato.sase.event_sub_type: "{{json_event.output.event_sub_type}}"
           cato.sase.custom_category: "{{json_event.output.custom_category}}"
+
+  set_ips_fields:
+    actions:
+      - set:
+          threat.indicator.ip: "{{json_event.output.src_ip}}"
+          threat.indicator.type: "ip"
+          threat.indicator.description: "{{json_event.output.threat_name}}"
+          threat.indicator.name: "{{json_event.output.src_ip}}"
+          threat.indicator.reference: "{{json_event.output.src_ip}}"
+
+      - set:
+          threat.tactic.id:
+            [
+              "{{parse_mitre_tactics.message.tactic_id_1}}",
+              "{{parse_mitre_tactics.message.tactic_id_2}}",
+            ]
+          threat.tactic.name:
+            [
+              "{{parse_mitre_tactics.message.tactic_name_1}}",
+              "{{parse_mitre_tactics.message.tactic_name_2}}",
+            ]
+        filter: "{{parse_mitre_tactics.message != None}}"
+
+      - set:
+          threat.technique.id:
+            [
+              "{{parse_mitre_techniques.message.technique_id_1}}",
+              "{{parse_mitre_techniques.message.technique_id_2}}",
+            ]
+          threat.technique.name:
+            [
+              "{{parse_mitre_techniques.message.technique_name_1}}",
+              "{{parse_mitre_techniques.message.technique_name_2}}",
+            ]
+        filter: "{{parse_mitre_techniques.message != None}}"
```

Comment:
I think we should also define `threat.technique.*` and `threat.tactic.*` for the `set_ng_malware_field` stage